### PR TITLE
NAS-109171 / 21.02 / Fix replication task table in scale

### PR DESCRIPTION
**Pull request by dszidi**

### CODE REVIEW

**Review by kmoore134 — APPROVED** [review]
Did a functional review here, confirmed this works and restores the lost button behavior.

File: src/app/pages/task-calendar/replication/replication-list/replication-list.component.ts [hunk 1]
```diff
@@ -35,8 +35,10 @@ export class ReplicationListComponent {
         { name: 'Target Dataset', prop: 'target_dataset', hidden: true},
         { name: 'Recursive', prop: 'recursive', hidden: true},
         { name: 'Auto', prop: 'auto', hidden: true},
-        { name: 'Enabled', prop: 'enabled', selectable: true },
-        { name: 'State', prop: 'task_state', state: 'state' },
+        //{ name: 'Enabled', prop: 'enabled', selectable: true },
```
Comment (denysbutenko) [inline src/app/pages/task-calendar/replication/replication-list/replication-list.component.ts:38]: Do we need to keep these comments? Line 38 and 40

**Review by denysbutenko — APPROVED** [review]
LGTM

File: src/app/pages/task-calendar/replication/replication-list/replication-list.component.ts (hunk 1 above)
Comment (dszidi) [reply src/app/pages/task-calendar/replication/replication-list/replication-list.component.ts]: Good catch. Those comments weren't supposed to be there. They've been removed.

**Pull request merged**